Docs/02-getting-started/quick-add-mtr-test.md

```diff
@@ -0,0 +1,99 @@
+---
+id: quick-add-mtr-test
+sidebar_position: 3.6
+---
+
+# Add and Run a MTR test
```

Comment:
The first header is used to render the page title. The following code is recommended instead：
```
# Qucik Add MTR Test

## Add and Run a MTR test

### ......
```